modules/cnv_install_1_4.adoc

```diff
@@ -0,0 +1,53 @@
+[[install_cnv_apb]]
+=== Installing {ProductName} to {product-title}
+
+The `kubevirt-ansible` RPM contains the latest automation for deploying 
+{ProductName}. This procedure installs all {ProductName} components to your 
+{product-title} cluster. 
+
+This procedure installs the following components:
+
+* {ProductName} core components (KubeVirt)
+* Containerized data importer (CDI) controller
+* Multus and Open vSwitch (OVS) container network interface plug-ins
+* Updated {product-title} web console
+
+.Prerequisites
+* {product-title} 3.11 cluster, running
+* User with `cluster-admin` privileges
+* xref:../apb_devel/cli_tooling.adoc#apb-devel-cli[APB tool]
+* `virtctl` client utility, installed
+
+.Procedure
+
+. Install the `kubevirt-ansible` RPM and its dependencies:
++
+----
+$ yum install kubevirt-ansible
+----
+
+. Navigate to the `kubevirt-ansible` directory:
++
+----
+$ cd /usr/share/ansible/kubevirt-ansible
+----
+
+. Edit `cnv.yml` to configure `kubevirt-ansible`:
++
+----
+$ vim vars/cnv.yml 
+----
++
+// add an example here!
+
+. Ensure that you are authenticated locally to the running OpenShift cluster.
+
+. Launch {ProductName} using the configuration file you edited earlier:
++
+----
+$ ansible-playbook -e @vars/cnv.yml --connection=local playbooks/kubevirt.yml
```

Comment:
@ousleyp please do not document as default practice to use `--connection=local` (cc @nellyc). You can mention it as a possible parameter but I'd like the common practice to be remote (as openshift-ansible).

Reply:
Sure, I will remove that.